newIDE/electron-app/app/main.js

```diff
@@ -12,12 +12,12 @@ const autoUpdater = require('electron-updater').autoUpdater;
 const log = require('electron-log');
 const {
   uploadGameFolderToBucket,
-  uploadArchiveToBucket,
+  uploadArchiveToBucket
```

Comment:
This file has all the comma removed, probably because you runned Prettier with a different trailingComma setup: https://prettier.io/docs/en/configuration.html

Can you fix these changes? they should not be included in the PR.

Reply:
ah, sorry. I need to update my prettier setting